Modules/Brand/Routes/api.php

```diff
@@ -13,31 +14,23 @@
 |
 */
 
-Route::prefix('brand')->group(function () {
-    Route::post('/register', 'BrandController@register');
-});
-Route::prefix('brand')->group(function () {
-    Route::post('/create', 'BrandController@create');
-});
-Route::prefix('brand')->group(function () {
-    Route::get('/edit/{id}', 'BrandController@edit');
-});
-Route::prefix('brand')->group(function () {
-    Route::post('/update-account', 'BrandController@updateAccount');
-});
-Route::prefix('brand')->group(function () {
-    Route::post('/update-shop', 'BrandController@updateShop');
-});
-Route::prefix('brand')->group(function () {
-    Route::post('/golive', 'BrandController@goLive');
-});
-Route::prefix('brand')->group(function () {
-    Route::get('/all/{retailer}', 'BrandController@all');
-});
-Route::prefix('brand')->group(function () {
-    Route::get('/customers', 'BrandController@customers');
-});
-Route::prefix('brand')->group(function () {
-    Route::post('/add-customer', 'BrandController@addCustomer');
-});
+Route::prefix('brands')
```

Comment:
Shouldn't this be under auth middleware?

Reply:
during registration of brand due to many steps some routes are accessed without login